tests/playwright/src/specs/image-manifest-smoke.spec.ts

```diff
@@ -0,0 +1,144 @@
+/**********************************************************************
+ * Copyright (C) 2025 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import path from 'node:path';
+import { fileURLToPath } from 'node:url';
+
+import { ArchitectureType } from '../model/core/platforms';
+import type { ImagesPage } from '../model/pages/images-page';
+import { expect as playExpect, test } from '../utility/fixtures';
+import { deleteImageManifest } from '../utility/operations';
+import { isCI, isWindows } from '../utility/platform';
+import { waitForPodmanMachineStartup } from '../utility/wait';
+
+const architectures: string[] = [ArchitectureType.AMD64, ArchitectureType.ARM64];
+const imageNameSimple: string = 'manifest-test-simple';
+const imageNameComplex: string = 'manifest-test-complex';
+const manifestNameSimple: string = `localhost/${imageNameSimple} (manifest)`;
+const manifestNameComplex: string = `localhost/${imageNameComplex} (manifest)`;
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+let imagesPage: ImagesPage;
+let skipTests: boolean = false;
+
+const hypervisor = process.env.GHA_HYPERVISOR ?? '';
```

Comment:
just thinking about using other approach, the VM type label should be present on podman machine, right? 
Besides, `CONTAINERS_MACHINE_PROVIDER` is an env. var. that can help you with what hypervisor is used.